src/jit/importer.cpp

```diff
@@ -792,7 +787,7 @@ void                Compiler::impAssignTempGen(unsigned     tmpNum,
 {
     GenTreePtr asg;
 
-    if (varTypeIsStruct(val))
+    if ((val->TypeGet() == TYP_STRUCT) || varTypeIsSIMD(val))
```

Comment:
It looks like this change is made to make sure the AVX (SIMD32) are excluded in the branch. Just wanted to make sure this is the reason (since the AVX support is not added yet).